Some assert updates.

davschneller opened the pull request:
Just a small patch which replaces all remaining libc asserts by our own ones.

davschneller committed:
Some assert updates.

davschneller commented:
Some files got modified with no change by accident... Encoding issue?

davschneller committed:
Clang fix (hopefully) for missing asserts.

davschneller committed:
Master merge for assert update.

k-lukas reviewed (approved):
Looks good.
I noticed, that you changed the line endings of some files from CRLF to LF. Maybe that could be done for alle remaining files in the project.

In this branch, the following files are still affected:
./spec/initial/arch/sections/impl.tex: LaTeX document, UTF-8 Unicode text, with very long lines, with CRLF line terminators
./source/ui/Components/editor/Editor.qml: ASCII text, with CRLF line terminators
./source/ui/Components/editor/IssueMark.qml: ASCII text, with CRLF line terminators
./source/ui/Components/editor/MacroTriangleButton.qml: ASCII text, with CRLF line terminators
./source/ui/Components/editor/MacroSubeditor.qml: ASCII text, with CRLF line terminators
./source/ui/ProjectCreationScreen.qml: ASCII text, with CRLF line terminators

davschneller committed:
Fixing some line ending-related errors.
(and providing a probably almost pointless script which helped to fix the errors)

davschneller commented:
Thanks for the review! The files should be re-encoded by now.
I'll merge the issue as soon as the tests are completed.
